lib/routes/utils/request_helper.js

```diff
@@ -80,8 +82,61 @@ function shouldSendVerifyAccountEmail(account, request) {
   return (!context || !!sendEmailIfUnverified) && !account.emailVerified
 }
 
+/**
+ * Returns whether or not to use signin unblock feature on a request.
+ *
+ * @param account
+ * @param config
+ * @param request
+ * @returns {boolean}
+ */
+function shouldEnableSigninUnblock(account, config, request) {
+
+  var enabled = config.signinUnblock && config.signinUnblock.enabled
+  if (!enabled) {
+    return false
+  }
+
+  //if (request.app.isSuspiciousRequest) {
+  //  return true
+  //}
```

Comment:
I'm pretty confident we can just delete this, anyone asking about sign-in unblock is already in the middle of a suspicious request :-)